test/language/expressions/template-literal/invalid-unicode-escape-sequence-1.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2016 Tim Disney. All rights reserved.
```

Comment:
This is a [file renaming](test/language/expressions/template-literal/invalid-unicode-escape-sequence-truncated.js), the copyrights should be kept in this specific case.